### ISSUE TITLE
Analogue Touch Stick Vibration

### ISSUE THREAD

**Issue by OpticTeadrop** [seleuco/MAME4droid-2024#70]
Hello, I have a question. 

In MAME4droid (0.139u1) the touch stick set to the 'analogue' mode used to vibrate with the "Vibrate" option enabled, which is no longer the case in the newest version of MAME4droid. That's a shame, as I really liked how it felt.

Is this behaviour intended? If not, could you restore how it worked in the old version and if it is, could you make it an option? Thanks

**Comment by seleuco** [comment]
In the old version it was actually vibrating due to a bug since in analog mode the digital mode was also being sent. In this version only analog data is sent. For analog mode I had to implement it and I haven't done it.

**Comment by OpticTeadrop** [comment]
Thaks for the quick reply!

Interesting how it happened due to a bug as the effect was really nice, kinda like a 'clickwheel' honestly. It being unintended probably means it would be hard to reimplement though. Oh well, thank you for replying anyways!

**Comment by seleuco** [comment]
Yeah. Really it was a fake digital vibration. You're right, it looked good enought. If I ever review that part I will think of something similar.
